bootstrap/digitalocean_k8s_ubuntu_16.04_master.sh

```diff
@@ -43,8 +43,8 @@ systemctl daemon-reload
 systemctl restart kubelet
 
 # Parse kubicorn configuration file.
-TOKEN=$(cat /etc/kubicorn/cluster.json | jq -r '.values.itemMap.INJECTEDTOKEN')
```

Comment:
Idea for another PR: above we have ${KUBERNETES_VERSION} stuff. Could we use Cluster API or API Machinery structures instead of those variables?

Reply:
yes we can make a lot of improvements here